src/HexagonOffload.cpp

```diff
@@ -60,7 +60,7 @@ Stmt replace_params(Stmt s, const std::map<std::string, Parameter> &replacements
 class InjectHexagonRpc : public IRMutator {
     std::map<std::string, Expr> state_vars;
 
-    Module device_code;
+    Module &device_code;
```

Comment:
Really I want it to return a Module, but it seemed strange to pass the module name in just so it could be constructed inside the visitor class. Using a reference was intended to make it clear the routine is filling in the Module and that this is its main result.